KenticoOnboardingApplication/test/KenticoOnboardingApplication.Tests.Base/Comparers/RetrievedItemComparer.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using KenticoOnboardingApplication.Contracts.Models;
+using KenticoOnboardingApplication.Contracts.Services.Wrappers;
+
+namespace KenticoOnboardingApplication.Tests.Base.Comparers
+{
+    public sealed class RetrievedItemComparer : IEqualityComparer<RetrievedItem<Item>>
+    {
+        private static readonly ItemComparer s_itemComparer = new ItemComparer();
```

Comment:
Having three is the new two, right? ([reference](https://www.irishexaminer.com/lifestyle/healthandlife/why-having-three-children-is-the-new-two-458513.html)) 😆

Reply:
But 👍 for reusing the `ItemComparer` class! 😃